docs/device_api.md

```diff
@@ -14,9 +14,69 @@ working as a client SDK for any desktop PC, laptop or gateway supporting a pytho
 the steps described below and use the python commands as described in the README.md file available at this [Github
 repository](https://github.com/telefonicaid/fiware-figway).
 
+# Configure the South Bound protocol
+
+In order for the South Bound protocols to work for your services, you must provision the information about your
+devices, either by provisioning the device itself, by provisioning a Configuration Group (for each subservice) or both.
+Configuration Groups (also called Services; do not mistake them for the multitenancy Service objects), define some
+default values for the South Bound protocol to NGSI mapping, that will be applied to every device associated to the group.
+Devices will be associated to groups based on the API Key provided by the Device in the communications.
+
+Configuration Groups can be provisioned through the API, or through the Provision Portal. If you are not the administrator
+of your subservices, you may have been given an API Key. If that's the case, there is no need to configure the
+South Bound protocol again. Use the provided data for future interactions.
+
+The following excerpt shows you how to provision a Configuration Group directly to the API. This request must be sent
+to the `/iot/services` path in an HTTP POST:
```

Comment:
Myabe the POST line should be included along with Fiware-Service and Fiware-ServicePath headers? Similar to https://github.com/telefonicaid/fiware-iot-stack/pull/44/files#diff-a5a5dc6c929e1c0f32b40cec24660d20R19